src/ClapCommandLineHandler.package/ClapCommandLineHandler.class/instance/matchCommandName..st

```diff
@@ -0,0 +1,4 @@
+as yet unclassified
+matchCommandName: arguments
+"This method matches the names of all the clapCommand handlers with the first arguments from given set of arguments."
+	^ self handlers select:  [:each | (each command canonicalName = arguments first asString) ].
```

Comment:
Here I disagree on design, because what this method does is already a form of parsing.
I think we already discussed that: my idea was for the command line handler to own or be itself a kind of _anonymous root command_, of which all actual clap commands would become subcommands (via pragmas or some other registration mechanism), instead of trying to find which command is concerned and only then starting the parse process.